app/components/preprint-form-authors.js

```diff
@@ -10,26 +10,158 @@ export default CpPanelBodyComponent.extend(PreprintFormFieldMixin, {
     permissionChanges: {},
     bibliographicChanges: {},
     valid: Ember.computed.alias('newContributorId'),
+    permissionToggle: false,
+    bibliographicToggle: false,
+    removalToggle: false,
+    stillAdmin: Ember.computed('isAdmin', function() {
+        return this.get('isAdmin');
+    }),
+    addState: 'emptyView', // There are 3 view states on left side of Authors panel. Default state just shows search bar.
+    query: null,
+    // Total contributor search results
+    totalSearchResults: Ember.computed('searchResults.[]', function() {
+        let searchResults = this.get('searchResults');
+        if (searchResults && searchResults.links) {
+            return Math.ceil(searchResults.links.meta.total);
```

Comment:
Why do we call `Math.ceil`? Isn't the total number of results already an integer?